src/widgets/RecommendationsPaintedDoorBtn/PaintedDoorExperimentContext.jsx

```diff
@@ -68,7 +68,11 @@ export const useIsEnterpriseUser = () => {
   return enterpriseUser;
 };
 
-export const PaintedDoorExperimentContext = React.createContext();
+export const PaintedDoorExperimentContext = React.createContext({
+  experimentVariation: null,
+  isPaintedDoorWidgetBtnVariation: null,
+  experimentLoading: null,
```

Comment:
```suggestion
    experimentVariation: null,
    isPaintedDoorNavbarBtnVariation: null,
    isPaintedDoorWidgetBtnVariation: null,
    isPaintedDoorControlVariation: null,
    experimentLoading: null,
```
Missed a few attributes, suggested above!